src/MIDebugEngine/Engine.Impl/Breakpoints.cs

```diff
@@ -325,5 +338,25 @@ internal AD7DocumentContext DocumentContext(AD7Engine engine)
 
             return new AD7DocumentContext(_textPosition, new AD7MemoryAddress(engine, Addr, this.FunctionName));
         }
+
+        /// <summary>
+        /// Returns the start line of the breakpoint.
+        /// NOTE: If set this overwrites any column or multiline information.
```

Comment:
Potentially the break-insert could have provided a range of information, however in practice I haven't seen it happen for gdb.